### ISSUE TITLE
Trash Icon not working properly

### ISSUE THREAD

**Issue by marczuverink** [micheleg/dash-to-dock#1143]
No right click empty.
Fedora 32
Gnome 3.36

**Comment by philipl** [comment]
You'll only see an Empty action in the menu if the trash has something in it.

**Comment by somepaulo** [comment]
Full Trash bin. Still no option to empty on right click.
Arch with GS 3.36.1

**Comment by philipl** [comment]
Are you using the 3.36 branch of dash to dock? Master doesn't work.

**Comment by somepaulo** [comment]
> Are you using the 3.36 branch of dash to dock? Master doesn't work.

Yes, I am. With Gnome on X, not Wayland.

**Comment by asegarra** [comment]
Same here, I noticed you get these warnings when you enable dash to dock

```
Apr 29 11:37:27 titan.lan gnome-shell[2505]: Impossible to monitor trash: Gio.IOErrorEnum: Operation not supported
Apr 29 11:37:27 titan.lan gnome-shell[2505]: Impossible to enumerate trash children: Gio.IOErrorEnum: Operation not supported
```

**Comment by lbrfabio** [comment]
Executing `sudo setcap -r /usr/bin/gnome-shell` resolve the problem but it has to be done at every gnome-shell update

**Comment by smschick** [comment]
Same issue where the trash is not empty and get no option to empty trash.  Installed via recommended https://extensions.gnome.org/extension/307/dash-to-dock/
GNOME version 3.36.3
Pop!_OS 20.04 LTS

**Comment by goncalossilva** [comment]
Same problem on Fedora 32. `sudo setcap -r /usr/bin/gnome-shell` does temporarily address it.

**Comment by proninyaroslav** [comment]
Yes, `sudo setcap -r /usr/bin/gnome-shell` works for me too (Fedora 33).
